src/main/kotlin/racingcar/domain/car/CarNames.kt

```diff
@@ -0,0 +1,6 @@
+package racingcar.domain.car
+
+data class CarNames(private val names: List<String>) {
+
+    fun mapToCar(): List<Car> = names.map { name -> Car(name) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    fun mapToCar(): List<Car> = names.map(::Car)
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/6ec9cfb2eef9750349d18f8eda60a0ccc20b9bc5 좋습니다!